flow/scripts/odb2files.tcl

```diff
@@ -0,0 +1,8 @@
+read_db $::env(ODB_FILE)
+write_def $::env(DEF_FILE)
+
+if { [info exists ::env(VERILOG_FILE)] } {
+    write_verilog $::env(VERILOG_FILE)
+}
```

Comment:
I think the `if`statement is not needed here, when the variable does not exist the script would fail silently.